src/server/api/controllers/profile.controller.js

```diff
@@ -18,7 +18,28 @@ const getStudentsProfile = async (userId) => {
     return error.message;
   }
 };
+const getMentorsProfile = async (userId) => {
+  try {
+    const profiles = await knex('users')
+      .select(
+        'users.name',
+        'users.timezone',
+        'organization_id',
+        'organizations.name as organizationName',
+        'skills.name as Skills',
```

Comment:
Did you test this? I find it hard to believe that this is an array. You should probably get skills in a new knex call and you should return numbers:
> skills: array of numbers from mentor_skills table

Also, it should be named `skills` with lowercase `s`.